### PR TITLE
chore: enable kubectl_wrapper_example to run on TPG4

### PR DESCRIPTION
**Pull request by apeabody**

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
main.tf
--- old/main.tf
+++ new/main.tf
@@ -130,8 +130,8 @@
   count = (var.enabled && !local.skip_download && var.platform == "linux") ? 1 : 0
 
   triggers = merge({
-    md5                 = md5(var.create_cmd_entrypoint)
-    arguments           = md5(var.create_cmd_body)
+    md5                    = md5(var.create_cmd_entrypoint)
+    arguments              = md5(var.create_cmd_body)
     download_numPy_command = local.download_numPy_command
   }, var.create_cmd_triggers)
 
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Review by bharathkkb — COMMENTED** [review]
@apeabody Thanks for the PR. This seems like something suited for the a script called through the gcloud module rather than as a built in. Any issues adding this to an existing script?

**Comment by apeabody** [comment]
> @apeabody Thanks for the PR. This seems like something suited for the a script called through the gcloud module rather than as a built in. Any issues adding this to an existing script?

Hmm, @bharathkkb  the issue is it's gcloud that is asking for numPy in some cases with IAP (`To increase the performance of the tunnel, consider installing NumPy.`), so there isn't currently a script.  I suppose we could wrap gcloud with a script to install numPy before the calls with`--tunnel-through-iap`, but that seems a little strange.  Let me know what you think.

**Comment by bharathkkb** [comment]
@apeabody I was suggesting we do the install when invoking the gcloud module. [Here is an example ](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/a0303606209234179ee0a480f8a91c6073301555/modules/asm/scripts/install_asm.sh#L105)where we wanted asmcli and we installed it via create_command.

**Comment by apeabody** [comment]
> @apeabody I was suggesting we do the install when invoking the gcloud module. [Here is an example ](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/a0303606209234179ee0a480f8a91c6073301555/modules/asm/scripts/install_asm.sh#L105)where we wanted asmcli and we installed it via create_command.

@bharathkkb - SG, stripping this PR to just the updates.

File: examples/kubectl_wrapper_example/main.tf [hunk 1]
```diff
@@ -20,7 +20,7 @@ locals {
 
 module "enabled_google_apis" {
   source  = "terraform-google-modules/project-factory/google//modules/project_services"
-  version = "~> 10.2"
+  version = "~> 13"
```
Comment (bharathkkb) [inline examples/kubectl_wrapper_example/main.tf:23]: nit: Lets pin to a minor version
```suggestion
  version = "~> 13.0"
```

**Review by bharathkkb — APPROVED** [review]

File: examples/kubectl_wrapper_example/main.tf (hunk 1 above)
Comment (apeabody) [reply examples/kubectl_wrapper_example/main.tf]: Thanks, didn't catch that when I updated the module version.

**Pull request merged**